ipaullly/sendIT
# [#161955974] Adds function to check for white spaces in the input data

#### What does this PR do?
Adds a function in the `utilities` directory to allow the view functions to check whether data from user contain white spaces or not
#### Description of Task to be completed?
- Define a function that removes the white spaces within the input data from the user and it returns a boolean
- Add a conditional in the POST register endpoint to allow only valid inputs to pass through to the database otherwise, it should throw an error message.
#### How should this be manually tested?
clone repo into a local directory
`git clone --single-branch -b bg-validate-whitespaces-161955974 git@github.com:ipaullly/sendIT.git`
cd into the sendIT directory
activate virtual environment
`source venv/bin/activate`
install dependencies
`pip install requirements.txt`
power up Postman and send JSON with a blank user email via a `POST register` endpoint to your server and you will see an error message that catches the invalid input format
